src/features/query-bar/components/filter/FilterButton.js

```diff
@@ -248,7 +269,9 @@ class FilterButton extends React.Component<Props, State> {
                                             areErrorsEnabled={areErrorsEnabled}
                                             index={index}
                                             columns={columns}
+                                            selectedConnector={selectedConnector}
                                             update={this.update}
+                                            updateselectedConnector={this.updateselectedConnector}
```

Comment:
This prop is named inconsistently.

QueryBar uses the "on XYZ Change" format. Similar to the "update" prop.